agent/utils/ephemeral_ports_test.go

```diff
@@ -144,7 +144,7 @@ func TestGetHostPortRange(t *testing.T) {
 					assert.Equal(t, tc.expectedLastAssignedPort[i], actualLastAssignedHostPort)
 				} else {
 					// need to reset the tracker to avoid getting data from previous test cases
-					tracker.SetLastAssignedHostPort(0)
+					ResetTracker()
```

Comment:
This function `ResetTracker()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.